consul/base.py

```diff
@@ -203,7 +203,8 @@ def json(
             one=False,
             decode=False,
             is_id=False,
-            index=False):
+            index=False,
+            txn=False):
```

Comment:
Is it possible to add new parameters to docs string?

Reply:
fixed